build-images.sh

```diff
@@ -18,7 +18,12 @@
 
 set -xe
 
-pyversion=${1:-"3.8.5"}
+if [ -z "$1" ];then
+  echo "Please supply python version to build image. Currently supported versions are 3.6, 3.7, 3.7.1, 3.8, and 3.8.5" >> /dev/stderr
```

Comment:
Probably better to suggest users to use help command to get supported versions, that will avoid future changes in this message.

Reply:
Since this is internal and only used by CircleCI, I'll remove the versions altogether.